packages/ember-htmlbars/tests/helpers/view_test.js

```diff
@@ -910,305 +242,6 @@ QUnit.test('{{view}} should be able to bind class names to truthy or falsy prope
-});
-
-QUnit.test('should bind to the property if no registered helper found for a mustache without parameters', function() {
-  view = EmberView.extend({
-    foobarProperty: computed(function() {
-      return 'foobarProperty';
-    })
-  }).create({
-    template: compile('{{view.foobarProperty}}')
-  });
-
-  runAppend(view);
-
-  ok(view.$().text() === 'foobarProperty', 'Property was bound to correctly');
-});
-
-QUnit.test('{{view}} should be able to point to a local instance of view', function() {
-  view = EmberView.create({
-    template: compile('{{view view.common}}'),
-
-    common: EmberView.create({
-      template: compile('common')
-    })
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'common', 'tries to look up view name locally');
-});
-
-QUnit.test('{{view}} should be able to point to a local instance of subclass of view', function() {
-  var MyView = EmberView.extend();
-  view = EmberView.create({
-    template: compile('{{view view.subclassed}}'),
-    subclassed: MyView.create({
-      template: compile('subclassed')
-    })
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'subclassed', 'tries to look up view name locally');
-});
-
-QUnit.test('{{view}} asserts that a view class is present', function() {
-  var MyView = EmberObject.extend();
-  view = EmberView.create({
-    template: compile('{{view view.notView}}'),
-    notView: MyView.extend({
-      template: compile('notView')
-    })
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /must be a subclass or an instance of Ember.View/);
-});
-
-QUnit.test('{{view}} asserts that a view class is present off controller', function() {
-  var MyView = EmberObject.extend();
-  view = EmberView.create({
-    template: compile('{{view notView}}'),
-    controller: EmberObject.create({
-      notView: MyView.extend({
-        template: compile('notView')
-      })
-    })
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /must be a subclass or an instance of Ember.View/);
-});
-
-QUnit.test('{{view}} asserts that a view instance is present', function() {
-  var MyView = EmberObject.extend();
-  view = EmberView.create({
-    template: compile('{{view view.notView}}'),
-    notView: MyView.create({
-      template: compile('notView')
-    })
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /must be a subclass or an instance of Ember.View/);
-});
-
-QUnit.test('{{view}} asserts that a view subclass instance is present off controller', function() {
-  var MyView = EmberObject.extend();
-  view = EmberView.create({
-    template: compile('{{view notView}}'),
-    controller: EmberObject.create({
-      notView: MyView.create({
-        template: compile('notView')
-      })
-    })
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /must be a subclass or an instance of Ember.View/);
-});
-
 QUnit.test('Specifying `id` to {{view}} is set on the view.', function() {
```

Comment:
Now covered in https://github.com/emberjs/ember.js/pull/13236/files#diff-c47395a5aaf92aa0a7edbfa812d4e0afR23